src/main/kotlin/racingcar/view/UserInterface.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.view
+import camp.nextstep.edu.missionutils.Console
+
+class UserInterface {
+    fun getCarName(): String {
+        println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+        val input = Console.readLine()
+        return input
+    }
+
+    fun getNumberOfAttempts(): String {
+        println("시도할 횟수는 몇 회인가요?")
+        val input = Console.readLine()
+        return input
+    }
+
+    fun showRaceStart() {
+        println()
+        println("실행 결과")
```

Comment:
`\n`으로 `println()`을 대체 할 수도 있습니다.
```suggestion
    fun showRaceStart() {
        println("\n실행 결과")
```